benchmark_runner/workloads/workloads_operations.py

```diff
@@ -77,6 +77,16 @@ def __init__(self):
         if self._enable_prometheus_snapshot:
             self._snapshot = PrometheusSnapshot(oc=self._oc, artifacts_path=self._run_artifacts_path, verbose=True)
 
+    def __get_workload_file_name(self, workload):
+            """
+            This method return workload name
```

Comment:
"returns"

Reply:
@RobertKrawitz, fix it. please let me know if there are more comments